lib/src/main/java/io/ably/lib/objects/LiveObjectsAdapter.java

```diff
@@ -0,0 +1,36 @@
+package io.ably.lib.objects;
+
+import io.ably.lib.plugins.PluginConnectionAdapter;
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.CompletionListener;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.ProtocolMessage;
+import io.ably.lib.util.Log;
+import org.jetbrains.annotations.NotNull;
+
+public interface LiveObjectsAdapter extends PluginConnectionAdapter {
+    void setChannelSerial(@NotNull String channelName, @NotNull String channelSerial);
+
+    class Adapter implements LiveObjectsAdapter {
```

Comment:
The `ConnectionManager` was becoming cluttered with unrelated methods, so a separate class was created to handle only liveobjects-related communication. This also improves readability by consolidating all liveobjects implementation in one place.